sdcm/nemesis.py

```diff
@@ -1547,7 +1547,9 @@ def disrupt_load_and_stream(self):
                                                                                         test_data=test_data)
             for sstables_info, load_on_node in map_files_to_node:
                 SstableLoadUtils.upload_sstables(load_on_node, test_data=sstables_info, table_name="standard1")
-                SstableLoadUtils.run_load_and_stream(load_on_node)
+                # NOTE: on K8S logs may appear with a delay, so add a bigger timeout for it.
```

Comment:
add a reference to the SCT issue, so when it's fixed we can drop this code.

Reply:
It is referenced in commit message.
Or you want to see it exactly in the code?

Reply:
yes in the code, so when fixing issue we can find all the workarounds easily